root/components/FormRowText.js

```diff
@@ -16,17 +16,18 @@ import FormLabel from './FormLabel';
 type Props = {
   +field: ReadOnlyFieldT<?string>,
   +label: string,
+  +onChange?: (SyntheticEvent<HTMLInputElement>) => void,
   +required?: boolean,
+  +size?: number,
   +type?: string,
-  ...
 };
 
 const FormRowText = ({
   field,
   label,
   required = false,
+  size,
   type = 'text',
-  ...inputProps
 }: Props) => (
```

Comment:
Props has onChange now, shouldn't this also be here then? What am I missing?

Reply:
I think you're right. It's fixed in the next commit, but I'll fix it here too.